### PR TITLE
Add test to publish content_view with multiple repos

### PR DESCRIPTION
**Pull request by kersommoura**
Add test to publish content_view with multiple RPM repos.

### CODE REVIEW

**Comment by kersommoura** [comment]
```python
PASSED2019-09-27 16:20:04 - robottelo - DEBUG - Finished Test: ContentViewPublishPromoteTestCase/test_positive_publish_multiple_repos
2019-09-27 16:20:04 - robottelo - INFO - Started tearDownClass: tests.foreman.api.test_contentview/ContentViewPublishPromoteTestCase


```

File: tests/foreman/api/test_contentview.py [hunk 1]
```diff
@@ -1180,6 +1181,35 @@ def test_positive_promote_out_of_sequence(self):
         self.assertEqual(len(content_view.version[0].read().environment), 1)
         self.assertEqual(len(content_view.version[-1].read().environment), 0)
 
+    @tier2
+    def test_positive_publish_multiple_repos(self):
+        """Attempt to publish a content view with multiple RPM repos.
```
Comment (vijay8451) [inline tests/foreman/api/test_contentview.py]:
```suggestion
        """Attempt to publish a content view with multiple yum repos.
``

File: tests/foreman/api/test_contentview.py [hunk 2]
```diff
@@ -1180,6 +1181,35 @@ def test_positive_promote_out_of_sequence(self):
...
+        for _ in range(10):
+            repo = entities.Repository(product=product).create()
+            repo.sync()
+            content_view.repository.append(repo)
+            content_view = content_view.update(['repository'])
+        content_view = content_view.read()
+        self.assertEqual(len(content_view.repository), 10)
+        with self.assertNotRaises(TaskFailedError):
+            content_view.publish()
+
```
Comment (vijay8451) [inline tests/foreman/api/test_contentview.py]: Could you please add assertion after publish as well ..
i.e.
```
self.assertEqual(len(content_view.read().version), 1)
```

Comment (kersommoura) [reply tests/foreman/api/test_contentview.py]: @vijay8451, thanks for your review. Updated per comments.

Comment (vijay8451) [reply tests/foreman/api/test_contentview.py]: thanks @kersommoura

**Comment by kersommoura** [comment]
@vijay8451, any other comment?

**Comment by mirekdlugosz** [comment]
@kersommoura Vijay already ACKed this PR, you want tier2 reviewer to ACK and merge. I requested their attention.

**Pull request merged**